python/py-skrf/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        scikit-rf scikit-rf 0.15.3 v
+
+checksums           rmd160  33172e881c9b13d464a3e4bf0a5329c60ac1b188 \
+                    sha256  c0f84010eec1ac07755c9fb8e51295a0116037f2de0a2117f507f7f1848106da \
+                    size    17777889
+
+name                py-skrf
+platforms           darwin
+maintainers         nomaintainer
+license             BSD
+
+description         scikit-rf (aka skrf): a Python package for RF engineering
+long_description    scikit-rf (aka skrf) is an Open Source, BSD-licensed \
+                    package for RF/Microwave engineering implemented in the \
+                    Python programming language. It provides a modern, \
+                    object-oriented library which is both flexible and scalable.
+
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append port:py${python.version}-setuptools
+}
```

Comment:
please make sure to add all required dependencies listed by [upstream](https://github.com/scikit-rf/scikit-rf/blob/247bc1452eec446019b2031fcf56d484268d1609/setup.py#L24)